packages/pebble-shared/svgs/Hidden.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000"><path d="M263.2 302.4l78 79.3c-37.2 27.6-67.2 64.6-85.9 107.6 41.2 94.2 135.4 160 244.9 160 31.4 0 61.6-5.5 89.6-15.5l89.9 91.1 27.6-27.2-416.6-422.6-27.5 27.3zM500 610.3c-84.9 0-162.1-47-201.6-121.1 17.1-32.2 41.6-59.2 70.4-79.6l39.2 39.7c-5.3 12.2-8.2 25.7-8.2 39.9 0 55.4 44.9 100.3 100.3 100.3 13.6 0 26.7-2.8 38.4-7.6l20.4 20.7c-19 5-38.8 7.7-58.9 7.7zm-60.7-129.2l67.9 69.1c-2.4.3-4.7.4-7.2.4-33.9 0-61.4-27.5-61.4-61.4 0-2.6.3-5.4.7-8.1zM592.2 528.8c5.1-12.1 8-25.4 8-39.5 0-55.4-44.9-100.3-100.3-100.3-13.4 0-26.4 2.6-38 7.5l31.2 31.8c2.2-.3 4.5-.4 6.8-.4 33.9 0 61.4 27.5 61.4 61.4 0 2.6-.3 5.1-.5 7.6l31.4 31.9z"/><path d="M500 368c84.9 0 162.1 47 201.6 121.1-17.1 32.1-41.2 59.1-70.1 79.5l27.6 28c37.1-27.6 67-64.6 85.7-107.4-41.4-94.2-135.3-160-244.8-160-31.4 0-61.4 5.4-89.3 15.3l30.8 31.2c18.9-4.9 38.5-7.7 58.5-7.7z"/></svg>
\ No newline at end of file
+<svg width="1000" height="1000" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)" fill="#000"><path d="M16.139 96.771l159.346 162.002C99.489 315.157 38.202 390.745 0 478.589c84.168 192.442 276.609 326.864 500.306 326.864 64.148 0 125.843-11.236 183.044-31.665l183.657 186.109 56.384-55.567L72.319 41l-56.18 55.771zM499.898 725.78c-173.442 0-331.154-96.016-411.849-247.395 34.934-65.781 84.985-120.94 143.82-162.615l80.082 81.103c-10.827 24.924-16.752 52.503-16.752 81.512 0 113.177 91.726 204.903 204.903 204.903 27.784 0 54.546-5.72 78.448-15.526l41.675 42.288c-38.815 10.214-79.265 15.73-120.327 15.73zM375.894 461.838l138.713 141.164c-4.903.613-9.602.817-14.709.817-69.254 0-125.434-56.18-125.434-125.434 0-5.311.613-11.032 1.43-16.547zM688.253 559.284c10.418-24.72 16.343-51.89 16.343-80.695 0-113.177-91.727-204.903-204.903-204.903-27.375 0-53.933 5.312-77.631 15.322l63.739 64.964c4.494-.613 9.193-.817 13.892-.817 69.254 0 125.434 56.18 125.434 125.434 0 5.312-.613 10.419-1.022 15.526l64.148 65.169z"/><path d="M499.898 230.786c173.442 0 331.154 96.016 411.849 247.395-34.934 65.577-84.168 120.735-143.208 162.41l56.384 57.202C900.715 641.409 961.798 565.821 1000 478.385c-84.576-192.441-276.405-326.864-500.102-326.864-64.147 0-125.434 11.032-182.431 31.256l62.921 63.739c38.611-10.01 78.652-15.73 119.51-15.73z"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h1000v1000H0z"/></clipPath></defs></svg>
```

Comment:
avoid using `clip-path` in svg. it doesnt work great on android ([see here](https://youtu.be/fgbl34me3kk?t=346))